components/tikv_alloc/Cargo.toml

```diff
@@ -11,7 +11,7 @@ jemalloc = ["tikv-jemallocator", "tikv-jemalloc-ctl", "tikv-jemalloc-sys"]
 # Build jemalloc's profiling features. Without this
 # certain profile functions will return nothing.
 mem-profiling = ["tikv-jemallocator/profiling", "log"]
-mimalloc = ["mimallocator"]
+snmalloc = ["snmalloc-rs"]
```

Comment:
Why the mimalloc feature is directly removed...

Reply:
because this is an optional dependency `mimalloc` which already gives the feature.

Reply:
because this is an optional dependency `mimalloc` which already gives the feature.

Reply:
or if you like I might be able to rename the dependency and rebind `mimalloc` feature flag to it.

Reply:
oh never mine. I see the dependency name is changed to `mimalloc` which is just identical to the feature name.